starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,60 @@
+* {
+  font-family: 'Open Sans', sans-serif;
+  text-align: justify;
+  font-weight: lighter;
+}
+
+h1 {
+  font-size: 70px;
+  text-align: center;
+  line-height: 600px;
+  letter-spacing: 5px;
+  color: white;
+  height: 600px;
+  background-size: 1000px 600px;
+  background-image: url('/Users/briannacho/code/labs/lab-css-recipes-clone/starter-code/apple-pie-starter-code/images/apple-pie.jpg');
```

Comment:
With this path, it only works on your computer (as you are using a path that start at the roots of your computer).

Try this instead : url('../images/apple-pie.jpg')
It will work on every computer as you are only refering to the folder path :)